include/s3select_functions.h

```diff
@@ -846,6 +855,119 @@ struct _fn_version : public base_function
   }
 };
 
+struct _fn_isnull : public base_function
+{
+
+  value res;
+  
+  bool operator()(bs_stmt_vec_t* args, variable* result)
+  {
+    bs_stmt_vec_t::iterator iter = args->begin();
+    base_statement* expr = *iter;
+    value expr_val = expr->eval();
+    if ( expr_val.is_null()) {
+      result->set_value((int64_t)true);
+    } else {
+      result->set_value((int64_t)false);
+    }
+    return true;
+  }
+};
+
+struct _fn_in : public base_function
+{
+
+  value res;
+  
+  bool operator()(bs_stmt_vec_t* args, variable* result)
+  {
+    bs_stmt_vec_t::iterator iter = args->begin();
+    bs_stmt_vec_t::iterator it = args->end();
+    base_statement* main_expr = *it;
+    value main_expr_val = main_expr->eval();
+    int args_size = args->size();
+    while (args_size > 1)
+    {
+      base_statement* expr = *iter;
+      value expr_val = expr->eval();
+      iter++;
+      if ( expr_val.type == main_expr_val.type) {
+        if ( expr_val == main_expr_val) {
+          result->set_value((int64_t)true);
+          return true;
+        } 
+    } 
+      args_size--;
+    }
+    result->set_value((int64_t)false);
+    return true;
+  }
+};
+
+struct _fn_like : public base_function
+{
+
+  value res;
+
+  std::string transform(std::string s)
+  {
+    bool startswith = (s[0] == '%' && s[s.size()-1] != '%');
```

Comment:
looking good. 
i had tried several combinations, and results were OK per those runs.

there are endless combinations, we should seek for a good coverage for such an operator.